src/app/app.component.html

```diff
@@ -1,37 +1,24 @@
-<md-sidenav-container fullscreen>
+<div class="app-fullpage">
   <md-toolbar color="accent" class="mat-elevation-z6">
```

Comment:
I think we should make the toolbar sticky, thoughts? @ladyleet @feloy

Reply:
I'll look how to do this. Previous solution wasn't perfect because sidenav was on top of toolbar.
It seems that "md-sidenav-container fullscreen" is not usable with a toolbar

Reply:
![capture du 2017-10-03 19 43 28](https://user-images.githubusercontent.com/9973512/31139872-2bca04ce-a873-11e7-85c2-970e56d32c1f.png)

In fact the toolbar is already sticky, isn't it? At least on Chrome and Firefox on Linux

Reply:
Apologies, just tested again and it looks great! 👍